cli/running/running.go

```diff
@@ -570,15 +571,28 @@ func FillCtx(cliOpts *config.CliOpts, cmdCtx *cmdcontext.CmdCtx,
 	return nil
 }
 
+func FillDaemonCtx(cliOpts *config.CliOpts, cmdCtx *cmdcontext.CmdCtx) error {
+	logPath := filepath.Join(cliOpts.Daemon.LogDir, cliOpts.Daemon.LogFile)
```

Comment:
cliOpts.Daemon may be `nil` if tarantool.yaml exists but no `daemon section` described.

Reply:
Config logic was rewritten, now daemon has his own config file `tt_daemon.yaml`.